tests/lib/nm/ipv4_test.py

```diff
@@ -109,11 +109,14 @@ def test_get_info_with_ipv4_config(NM_mock):
     address_mock = mock.MagicMock()
     config_mock.get_addresses.return_value = [address_mock]
     remote_conn_mock = mock.MagicMock()
-    act_con_mock.get_connection().return_value = remote_conn_mock
+    act_con_mock.get_connection.return_value = remote_conn_mock
```

Comment:
This seems to be an unrelated bugfix. If it is not too much trouble, move it to a separate commit. Also the one in ipv6_test.